### ISSUE TITLE
`deplot.close` doesn't close the spawned window

### ISSUE THREAD

**Issue by rnbguy** [JOTSR/Deplot#1]
I am trying to use `deplot` as following.

```
const deplot = new Deplot("Plotly", {port: 8000});
const myPlot = deplot.plot(datas);
deplot.close(myPlot);
```

But it doesn't close the window. I want to close a previous plot and spawn a new one.

**Comment by rnbguy** [comment]
I have a feeling this case is not working properly.
https://github.com/JOTSR/Deplot/blob/188e2dd8cca31be01d151b773781842b00ae7d8f/src/deplot.ts#L140-L145

**Comment by JOTSR** [comment]
Thank you for your report.

The main cause of this issue is the blocking event loop of deno webview, which I used to render the UI. I used websocket to communicate with an embedded js controller in the webview frame but it don't work properly. Webview block the event loop so I can't directly handle the closing process from the server side even if I spaw the ui in a dedicated worker.

[Deno webview](https://github.com/webview/webview_deno/pulls) seems to be (at least partially) frozen, and I have some bug fix and feature PRs that have not been reviewed for few month, so I could not make any progress since last week. However, I found a new and very interesting project, [deno webui](https://github.com/webui-dev/deno-webui), that solves many problems of deno webview and is very active. I am currently working with the webui team to improve the deno port and the C library. I am waiting for the next release of the module to update deplot and switch from webview to webui (I will update deplot at least this week).

After the switch to webui, deplot will be more responsive and handle multiwindows better without blocking the main event loop. Refresh a plot with new data will also be added.
